### PR TITLE
Add hupu extension

### PR DESCRIPTION
**Pull request by alpacachen**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by alpacachen** [comment]
hupu is  china sports news website
There are many loyal users

**Review by alpacachen — COMMENTED** [review]
im not native english speaker

File: extensions/hupu/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Raycast Search Hupu
+
+通过Raycast来浏览[虎扑步行街](https://bbs.hupu.com/all-gambia)!
```
Comment (pernielsentikaer) [inline extensions/hupu/README.md:3]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

Comment (alpacachen) [reply extensions/hupu/README.md]: done

**Pull request merged**